src/main/java/org/folio/inventory/resources/ItemStatusURL.java

```diff
@@ -0,0 +1,37 @@
+package org.folio.inventory.resources;
+
+import org.folio.inventory.domain.items.ItemStatusName;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Optional;
+import java.util.stream.Collectors;
+
+public class ItemStatusURL {
+  private static final Map<ItemStatusName, String> itemStatusNameUrlMap;
+  private static final Map<String, ItemStatusName> urlItemStatusNameMap;
+
+  static {
+    itemStatusNameUrlMap = new HashMap<>();
+
+    itemStatusNameUrlMap.put(ItemStatusName.IN_PROCESS, "/mark-in-process");
+    itemStatusNameUrlMap.put(ItemStatusName.IN_PROCESS_NON_REQUESTABLE, "/mark-in-process-non-requestable");
+//    itemStatusNameUrlMap.put(ItemStatusName.INTELLECTUAL_ITEM, "/mark-intellectual-item");
+//    itemStatusNameUrlMap.put(ItemStatusName.LONG_MISSING, "/mark-long-missing");
+//    itemStatusNameUrlMap.put(ItemStatusName.MISSING, "/mark-missing");
+//    itemStatusNameUrlMap.put(ItemStatusName.RESTRICTED, "/mark-restricted");
+//    itemStatusNameUrlMap.put(ItemStatusName.UNAVAILABLE, "/mark-unavailable");
+//    itemStatusNameUrlMap.put(ItemStatusName.UNKNOWN, "/mark-unknown");
+//    itemStatusNameUrlMap.put(ItemStatusName.WITHDRAWN, "/mark-withdrawn");
```

Comment:
As this pull request is intended to only cover the in process statuses, please remove this commented out code until those other statuses are implemented.

Reply:
I have been planned to implement other statuses after aproving of "comon idea" within this PR. This commented code is my "roadmap". 

2

Reply:
Ok, if that is going to be done in this pull request, then there is no need to remove it